tests/clients/test_experiment.py

```diff
@@ -121,6 +128,11 @@
 }
 
 
+def test_tag_schema():
+    data = TagSchema().load(TAG_BODY)
+    assert data == TAG
+
+
 def test_experiment_schema():
```

Comment:
Can we also test the case where non-empty tags are passed?

Reply:
I think this is now covered by the tests that Maitiu added and the one that I added.